### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by rustam-tolipov**
### Vet clinic database: create animals table

- Create an animals table. 💯
- Insert given data into the table. 💯
- Write queries for animals table. 💯

Feel free to review 🧑🏻‍💻 the PR and let me know if there are any changes to do! 🤓

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
```
Comment (DeliceLydia) [inline schema.sql]: ### Optional
- [x] I suggest that you make the ID column not null and autoincremented
like `id int NOT NULL IDENTITY`
- [x] Mark the id column as primary key
`PRIMARY KEY (id)`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals where date_of_birth BETWEEN DATE '2016-01-01' AND '2019-01-01';
+SELECT name, neutered, escape_attempts FROM  animals WHERE neutered = 'yes' AND escape_attempts < 3;
```
Comment (DeliceLydia) [inline queries.sql]: - [x] This query is about selecting names only of all animals that are neutered and have less than 3 escape attempts please update it

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+SELECT name FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
```
Comment (DeliceLydia) [inline queries.sql]: - [x] This query is about selecting the date of birth not names of all animals named either "Agumon" or "Pikachu" please fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Rustamxon7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (rustam-tolipov) [reply schema.sql]: Thanks for reminding me about these issues! I used BIGSERIAL(BIGINT) for generating ids.

File: queries.sql (hunk 2 above)
Comment (rustam-tolipov) [reply queries.sql]: I changed to list only names in this query!

File: queries.sql (hunk 3 above)
Comment (rustam-tolipov) [reply queries.sql]: This one will show only names in this query!

**Comment by Davidosky007** [comment]
## Hi @Rustamxon7 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 


CR GIF

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**